.github/workflows/ci.yml

```
@@ -23,7 +23,20 @@ jobs:
     name: Test Types with TypeScript ${{ matrix.ts }}
     strategy:
       matrix:
-        ts: ["4.7", "4.8", "4.9", "5.0", "5.1", "5.2", "5.3", "5.4", "5.5"]
+        ts:
+          [
+            "4.7",
+            "4.8",
+            "4.9",
+            "5.0",
+            "5.1",
+            "5.2",
+            "5.3",
+            "5.4",
+            "5.5",
+            "5.6",
+            "5.7",
+          ]
```

This is weird yaml. Good idea to make multiline at this point but should be 
```suggestion
        ts:
          - "4.7"
          - "4.8"
          - "4.9"
          - "5.0"
          - "5.1"
          - "5.2"
          - "5.3"
          - "5.4"
          - "5.5"
          - "5.6"
          - "5.7"
```